lib/matter_compiler/blueprint.rb

```diff
@@ -76,7 +76,13 @@ def load_ast!(ast)
       return if ast.empty?
       @collection = Array.new
       ast.each do |item|
-        @collection << Hash[item[:name].to_sym, item[:value]]
+        if item.has_key?("Authorization")
```

Comment:
@abtris what is it a legacy header? I am sorry but I am missing the sense here...